resources/views/gallery/edit.blade.php

```diff
@@ -104,6 +110,32 @@
     </div>
 </div>
 
+<div class="gallery-toggle collapse !visible">
+    <div class="modal-body panel-image-edit">
+        <div class="">
+            <x-box>
+                <p class="help-block">{{ __('entities/image.focus.helper') }}</p>
+                <div class="focus-selector max-h-screen relative mb-2 overflow-auto">
+                    <div class="focus absolute text-white cursor-pointer text-3xl" style="@if(empty($image->focus_x))display: none; @else left: {{ $image->focus_x }}px; top: {{ $image->focus_y }}px; @endif">
+                        <i class="fa-regular fa-bullseye fa-2x hover:text-red" aria-hidden="true"></i>
+                    </div>
+                    <img class="focus-image max-w-none" src="{{ $image->getImagePath(0) }}" alt="img" />
+                </div>
+                <div class="flex gap-2 sm:gap-5 items-center mb-5 pull-right">
```

Comment:
Don't think that `pull-right` is needed here, instead the first form can have a "grow" class to make it stretch, pushing the save to the right as from the ticket's mockup